testutils/common.py

```diff
@@ -320,47 +320,6 @@ def get_mender_artifact(
         os.path.exists(artifact) and os.unlink(artifact)
 
 
-def wait_for_traefik(gateway_host, routers=[]):
```

Comment:
Thanks for the clean-up. I guess this got obsolete when we introduced `wait_until_healthy`